Assets/Feature01/Scripts/AbstractClasses/CharacterAbstractClass.cs

```diff
@@ -0,0 +1,8 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public abstract class CharacterAbstractClass
+{
+    public abstract void characterInstantiate();
```

Comment:
Method Name will be CamelCase. 
First character will be capital. Need to follow one Standard so that everyone is on the same ground

Reply:
yes bhai update kortesi